ios/titanium-facebook.xcodeproj/project.pbxproj

```diff
@@ -343,7 +343,7 @@
 				GCC_WARN_UNUSED_VALUE = NO;
 				GCC_WARN_UNUSED_VARIABLE = NO;
 				INSTALL_PATH = /usr/local/lib;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```

Comment:
In titanium SDK deployment target is 10.0. Same should be here.

Reply:
Oh, I thought it's being bumped to 11 in SDK 10. But sure, will do!